crates/node/src/chain_spec.rs

```diff
@@ -80,6 +88,25 @@ pub fn development_config(enable_manual_seal: Option<bool>) -> Result<DevChainSp
     ))
 }
 
+// helper to print development accounts info
+// accounts with addresses 0x1 and 0x4 are NO VALIDATE accounts (don't require PK)
+// accounts with addresses 0x2 and 0x3 have the same PK
+pub fn print_development_accounts() {
+    let loader: GenesisLoader = serde_json::from_str(&STARKNET_GENESIS).unwrap();
+    let starknet_genesis: madara_runtime::pallet_starknet::GenesisConfig<Runtime> = loader.into();
+    let no_validate_account_address = starknet_genesis.contracts[0].0.0.to_string();
+    let argent_account_address = starknet_genesis.contracts[1].0.0.to_string();
+    let oz_account_address = starknet_genesis.contracts[2].0.0.to_string();
+    let cairo_1_no_validate_account_address = starknet_genesis.contracts[3].0.0.to_string();
+
+    const ARGENT_PK: &str = "0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d";
+    log::info!("🧪 Using the following development accounts:");
+    log::info!("🧪 NO VALIDATE with address: {} and pk: {}", no_validate_account_address, "");
```

Comment:
Remove the `and pk: {}` part. Or replace it with "and no pk"

Same on line 107

Reply:
done!